github-event-service/pull-request.ts

```diff
@@ -57,3 +51,20 @@ export async function handlePullRequestEvent(event: GithubPullRequestEvent) {
   }
 }
 
+async function createPresubmitTask(
+  repoId: string, pullRequestRef: firebase.firestore.DocumentReference) {
+    const repoDoc = await getConfig(repoId);
+    if (!repoDoc.exists) {
+      return;
+    }
+    const repoConfig = (await getConfig(repoId)).data() as BranchManagerRepoConfig;
+    if (!repoConfig.enabled) {
+      return;
+    }
+    const target = await getTargetBranch(repoId, pullRequestRef);
+    const pullRequest = (await pullRequestRef.get()).data() as BranchManagerPullRequest;
+    console.log(`sendpresubmittasktopubsub - ${pullRequest.pullRequestNumber}`);
```

Comment:
Is this following some logging scheme? Is there something that prevents camelCasing or hyphenating `sendpresubmittasktopubsub`?